### ISSUE TITLE
Normal objects with __call__ attr have their functionality obscured

### ISSUE THREAD

**Issue by weissjeffm** [RedHatQE/function_trace#4]
If a module has a variable that's an instance of a user defined class, and that class has `__call__` attr, after adding trace, the rest of the object's interface is then inaccessible.

**Comment by weissjeffm** [comment]
```
16:29 <jweiss> smyers: would you happen to know, when you do obj() what is the first attribute it looks up on obj?
16:29 <jweiss> if it's __call__ i think we're golden
16:29 <smyers> Possibly __call__
16:30 <smyers> But I don't know for sure
16:30 <jweiss> ok, i'll mess with it in ipython
```

**Comment by weissjeffm** [comment]
Fixed in 32e8dc640b5038af7f9c2fd00bc2a4d513f5523e
